service/oauth_service/find.go

```diff
@@ -9,5 +9,10 @@ type FindRequest struct {
 
 // Find returns the found user
 func (s *service) Find(req *FindRequest) (*model.User, error) {
-	return s.UserStore.FindByGithubUser(req.UUID)
+	return s.UserStore.FindByGithubUUID(req.UUID)
+}
+
+// FindByAuthToken returns user that found by given access token
+func (s *service) FindByAuthToken(token string) (*model.User, error) {
+	return s.UserStore.FindByAuthToken(token)
```

Comment:
Not correct.
Multiple access token can be issued for a single user and it's likely to be changed 
after time elapsed.

So it's non-sense to compare two access token to identify user.
You'll need to lookup GitHub's user id (UUID) by access token, and then find a user with that UUID.